code/__DEFINES/wiremod.dm

```diff
@@ -49,7 +49,7 @@
 // Components
 
 /// The value that is sent whenever a component is simply sending a signal. This can be anything.
-#define COMPONENT_SIGNAL 1
+#define COMPONENT_SIGNAL (world.time SECONDS)
```

Comment:
don't do world.time SECONDS, just do world.time * 10.

This looks incredibly confusing to someone who might not understand it, and it looks really ugly in general.

Reply:
would you want all uses of SECONDS everywhere removed, then?

Reply:
???
no. You don't do `variable SECONDS`, you do `10 SECONDS`.
You also don't divide by 10 because that converts it into miliseconds.

Reply:
Would `(world.time / (1 SECONDS))` as Mothblocks said be better?

Reply:
:shrug:

Reply:
`world.time SECONDS` is incorrect because `world.time` is *already* in deicseconds. `SECONDS` multiplies it by 10.